extensions/positron-r/src/provider.ts

```diff
@@ -89,6 +89,12 @@ export async function* rRuntimeDiscoverer(): AsyncGenerator<positron.LanguageRun
 	const userMoreBinaries = discoverAdHocBinaries(userRBinaries());
 	updateBinaries(userMoreBinaries);
 
+	// Directories relevant to Posit Workbench. Not relevant on Windows.
+	if (os.platform() !== 'win32') {
+		const pwbBinaries = discoverPWBBinaries();
```

Comment:
So we're still going to consult these locations on macOS. Are we sure we want to do that? I'm not necessarily for or against, but we should be deliberate.